server/events/project_command_builder.go

```diff
@@ -2,6 +2,10 @@ package events
 
 import (
 	"fmt"
+	"github.com/runatlantis/atlantis/server/fetchers"
```

Comment:
Mixing stdlib and non-stdlib imports is unidiomatic in Go. I recommend configuring your editor ro run `goimports` on save and it'll move it for you.